src/components/ForestTypeModal/ForestTypeDescription/so/ForestTypesDescription.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack';
+
+import Button from '../../../Button';
+
+const containerStyle = { margin: '-1.5rem', position: 'relative' };
+const buttonStyle = { position: 'absolute', top: 10, right: 10, zIndex: 999 };
+
+function ForestTypeDescription({ code }) {
+  const name = code.replace('*', 'stern');
+  const containerRef = useRef(null);
+  const [numPages, setNumPages] = useState(null);
+  const { t } = useTranslation();
+
+  const onDocumentLoadSuccess = ({ numPages: nextNumPages }) => {
+    setNumPages(nextNumPages);
+  };
+
+  return (
+    <div style={containerStyle} ref={containerRef}>
+      <Button
```

Comment:
There needs to be a little more margin between the export button and the PDF, it looks generally a bit cramped and on mobile the button actually overlaps and obscures the PDF title.

Reply:
I resolved this